cypress/e2e/CMS/$AppModule.feature

```diff
@@ -0,0 +1,80 @@
+Feature: As a cms user I want to be able to create Article content type to promote Article
+
+    Scenario: User is adding new article content type
```

Comment:
same - User is adding new App module content type

Reply:
fixed